Description: Rename ndg-httpsclient requirement as ndg_httpsclient so it can be found correctly during build.

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- conn-check-1.0.18.orig/requirements.txt
+++ conn-check-1.0.18/requirements.txt
@@ -1,6 +1,6 @@
 characteristic >= 14.0
 cryptography >= 0.5
-ndg-httpsclient
+ndg_httpsclient
 pyasn1
 pyOpenSSL
 service_identity
